NovemberQA/NovemberQA/Feature/Skill.feature

```diff
@@ -15,11 +15,17 @@ Examples:
 | Teaching | Beginner     |
 | PetCare  | Intermediate |
 
-Scenario: Edit Skills in the Profile
-    When I edit Skills and Level to profile
-	Then The Skills and Level should be edited to the profile successfully
+@editskills
+Scenario Outline: B2 Edit Skills in the Profile
+    When I edit '<Skills>' and '<Level>' to profile
+    Then The '<Skills>' and '<Level>' should be edited to the profile successfully
 
-Scenario: Delelte Skills in the Profile
+Examples: 
+| Skills  | Level    |
+| Writing | Beginner |
+
+@deleteskills
+Scenario: B3 Delelte Skills in the Profile
     When I delete Skills and Level from profile
```

Comment:
furture reference can add table here too, maybe for the tasks ahead you can add this